pkg/node/manager/manager.go

```diff
@@ -571,14 +547,12 @@ func (m *Manager) StartNeighborRefresh(nh datapath.NodeHandler) {
 						continue
 					}
 					go func(c context.Context, e nodeTypes.Node) {
-						n := randGen.Int63n(int64(interval / 2))
-						time.Sleep(interval/2 + time.Duration(n))
```

Comment:
I agree with the removal of the `net.ipv4.neigh.default.base_reachable_time_ms` based-interval, but this sleep statement here seems to introduce some randomness such that we don't ping all nodes at once, do we want to keep it?

Reply:
Yeah, I was thinking to add this in a separate PR to avoid overcomplicating this one so that reviewers could focus on concurrency aspects of this PR instead. But anyway, I will add it.